test/utils/test_hf.py

```diff
@@ -81,3 +85,90 @@ def test_convert_message_to_hf_invalid():
     )
     with pytest.raises(ValueError):
         convert_message_to_hf_format(message)
+
+
+def test_convert_message_to_hf_format_with_images():
+    # Test image with text message (image-only not supported by ChatMessage.from_user)
+    image = ImageContent(base64_image=TEST_IMAGE_PNG, mime_type="image/png", validation=False)
```

Comment:
Here and in the following tests, I would leave `validation=True` (default value)